docassemble/MotiontovacateDefaultJudgment/data/questions/Motion_to_Vacate.yml

```diff
@@ -6,7 +6,10 @@ code: |
   user_role = "defendant"
   other_parties.there_are_any = True
   basic_questions_intro_screen
-  remove_default_intro
+  if sign_agreement:
+    if default_agreement:
+      defaulted_after_agreement
+  # remove_default_intro
```

Comment:
Gave users choices that would kick them out at the **very** beginning. and added id: defaulted after agreement screen l. 74